vivarium/actor/process.py

```diff
@@ -10,6 +10,7 @@
 from vivarium.utils.dict_utils import merge_dicts, deep_merge
 
 
+GLOBAL_STATE_KEY = '__global__'
```

Comment:
I feel like there's a better name to be found for this variable and key. It stores a reference to the compartment object, but that's not communicated by "global". Maybe:

```suggestion
COMPARTMENT_STATE_KEY = 'compartment_state'
```

Reply:
I like 'compartment_state'.  We still need to choose terms for all the other kinds of states.

Reply:
Okay I like that too: https://github.com/CovertLab/vivarium/pull/135/commits/7542c2abba926d6b8f0abb12c2189bbbe4061613

Thanks for the feedback! Glad we arrived at a copacetic solution.